cluster/gce/util.sh

```diff
@@ -235,7 +235,7 @@ function copy-to-staging() {
     fi
   fi
 
-  #echo "${hash}" > "${tar}.sha512"
```

Comment:
This needs to be commented out. Bazel generates this file and it will generate permission denied error.